### ISSUE TITLE
include-pattern ruleset configuration is not working

### ISSUE THREAD

**Issue by neilime** [phpmd/phpmd#1187]
- PHPMD version: all
- PHP Version: all
- Installation type: all
- Operating System / Distribution & Version: all

### Current Behavior

The xsd schema specify an allowed node named "include-pattern" . But there is no occurence in the code where this node is supported

https://github.com/phpmd/phpmd/blob/caf04a5c1c91bdecefbe900c2a7d6235aa5c6106/src/site/resources/web/xml/ruleset_xml_schema_1.0.0.xsd#L13

### Expected Behavior

When "include-pattern" is set in the rule set it must be take into account and PHPMD must parse the files included byuu this pattern

### Steps To Reproduce:

- Create a project with :
```
index.php
src
   test.php
```

- phpmd.xml:
```xml
<ruleset name="PHPMD RuleSet"
    xmlns="http://pmd.sf.net/ruleset/1.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd" xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
    <description>PHPMD RuleSet</description>
    <include-pattern>src/*</include-pattern>
</rulset>
```

- run `phpmd . ansi phpmd.xml`

PHPMD will parse `index.php` while it is not in `src/`

## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "JSON rendering failed on Windows for filenames with space".

**Comment by LedzZm** [comment]
I don't see that option in the documentation?? Does that exist?

It would be really handy, that is for sure.

**Comment by neilime** [comment]
> I don't see that option in the documentation?? Does that exist?
> 
> It would be really handy, that is for sure.

At least it referneced in xml schema: https://github.com/phpmd/phpmd/blob/47bb51c63d9340fa27db67595f65e254a5bd3db3/src/site/resources/web/xml/ruleset_xml_schema_1.0.0.xsd#L13

**Comment by AJenbo** [comment]
Isn't this because you are expecting it to overwrite the argument but it's in addition?

**Comment by neilime** [comment]
I don't think so, there is no occurrence in the code where "include-pattern" can be used
